bom-latest/pom.xml

```
@@ -216,6 +234,11 @@
                 <artifactId>pipeline-input-step</artifactId>
                 <version>2.11</version>
             </dependency>
+            <dependency>
+                <groupId>org.jenkins-ci.plugins</groupId>
+                <artifactId>pipeline-stage-step</artifactId>
+                <version>2.4</version>
```

```suggestion
                <version>2.5</version>
```

to pick up https://github.com/jenkinsci/pipeline-stage-step-plugin/pull/22